### PR TITLE
Easy test Problem /8

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Contributes to #29136

### CODE REVIEW

File: aspnetcore/mvc/models/validation/samples/ValidationResultErrorMessage/ContactsController.cs [hunk 1]
```diff
@@ -1,164 +1,163 @@
...
-using System.Linq;
-using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc;
-using Microsoft.AspNetCore.Mvc.Rendering;
 using Microsoft.EntityFrameworkCore;
 using ValidationResultErrorMessage.Data;
 using ValidationResultErrorMessage.Models;
 
-namespace ValidationResultErrorMessage
+namespace ValidationResultErrorMessage;
```
Comment (Rick-Anderson) [inline aspnetcore/mvc/models/validation/samples/ValidationResultErrorMessage/ContactsController.cs]: We should do this on any code we write.
But it totally messes up GitHub diff.

File: aspnetcore/mvc/models/validation/samples/ValidationResultErrorMessage/ContactsController.cs [hunk 2]
```diff
@@ -151,14 +146,18 @@ public async Task<IActionResult> DeleteConfirmed(Guid id)
             {
                 _context.Contact.Remove(contact);
             }
-            
+            else
+            {
+                return Problem("Contact deleted.");
+            }
+
```
Comment (Rick-Anderson) [inline aspnetcore/mvc/models/validation/samples/ValidationResultErrorMessage/ContactsController.cs:153]: I added this to make it easy to test. Click open in new tab on delete button. Then delete the item. In new tab, try delete again.

**Review by Rick-Anderson — COMMENTED** [review]
This is a really great sample, but I'm wonder if we should convert it to a Web API app. You really don't want a UI app returning

{"type":"https://tools.ietf.org/html/rfc7231#section-6.6.1","title":"An error occurred while processing your request.","status":500,"detail":"Contact deleted.","traceId":"00-bfdefd61ac9c6f6548bc2d999ea2b685-28769c063ba71723-00"}

What do you think @sammychinedu2ky

File: aspnetcore/mvc/models/validation/samples/ValidationResultErrorMessage/ContactsController.cs (hunk 2 above)
Comment (sammychinedu2ky) [reply aspnetcore/mvc/models/validation/samples/ValidationResultErrorMessage/ContactsController.cs:153]: ok let me see

Comment (sammychinedu2ky) [reply aspnetcore/mvc/models/validation/samples/ValidationResultErrorMessage/ContactsController.cs:153]: I could return the Error View while serializing the JSON into the Error model.
image
@Rick-Anderson

**Pull request merged**